config/application.properties

```diff
@@ -24,4 +24,8 @@ amqp.messageConverter=io.strimzi.kafka.bridge.amqp.converter.AmqpDefaultMessageC
 #HTTP related settings
 http.enabled=true
 http.host=0.0.0.0
-http.port=8080
\ No newline at end of file
+http.port=8080
+#enable cors
+http.cors.enabled=false
+http.cors.allowedOrigins=https://strimzi.io
```

Comment:
why don't put just "*" in the properties files we have across the project. This value doesn't make sense to me.